### DISCUSSION TITLE
API POST /api/v1/images is not documented in the openapi

### DISCUSSION THREAD

**Discussion by Marcondiro** [artifacthub/hub#3482] [Q&A] [answered]
Hello, the API `POST /api/v1/images` is not documented in the openapi, I'm wondering if this is a choice or an oversight thx :)

**Comment by Marcondiro** [comment]
Also the `CSRF` apis appears to be missing

**Comment by tegioz** [comment, marked as answer]
Hi @Marcondiro

That was intentional, yes 🙂

**Reply by Marcondiro** [reply]
Alright, thanks 🙂
